### PR TITLE
cleaning imports

### PR DESCRIPTION
**Pull request by phpgeekfr**
movie some models to `****.firestore.ts` and updated `apps.ts`

As @GrandSchtroumpf suggested, I will create an issue to move all firestore.ts files into a new `models` lib

@RemcoSimonides can you try to deploy again with these changes ? It worked on my side  but just to be sure ..

### CODE REVIEW

**Comment by phpgeekfr** [comment]
issue https://github.com/blockframes/blockframes/issues/6771

**Comment by RemcoSimonides** [comment]
Deploy succesful :)

File: libs/utils/src/lib/apps.ts [hunk 1]
```diff
@@ -124,7 +122,7 @@ export function getAppName(slug: App, short = false) {
  * getOrgAppAccess(orgA); // ['catalog', 'festival']
  * getOrgAppAccess(orgB); // ['festival']
  */
-export function getOrgAppAccess(org: OrganizationDocument | Organization, first: App = 'festival'): App[] {
+export function getOrgAppAccess(org: OrganizationDocument | OrganizationBase<Date>, first: App = 'festival'): App[] {
```
Comment (RemcoSimonides) [inline libs/utils/src/lib/apps.ts:125]: It wasn't possible to move Organization to `.firestore.ts`? or would it break too much imports throughout the app?

**Review by RemcoSimonides — APPROVED** [review]

**Comment by phpgeekfr** [comment]
> Deploy succesful :)

Ho yeah ! I really don't get why you are the first with this issue. Problem was here for a long time

File: libs/contract/src/lib/bucket/+state/bucket.firestore.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { Holdback } from '@blockframes/contract/contract/+state/contract.firestore';
+import { BucketTerm } from '@blockframes/contract/term/+state/term.firestore';
+import { MovieCurrency } from '@blockframes/utils/static-model';
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/bucket/+state/bucket.firestore.ts:3]: Maybe be more precise here to not import all the static-model. Another option is to put `import type {...}` like that typescript knows if should remove this line at compile time.

File: libs/contract/src/lib/contract/+state/contract.firestore.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+
+import { DocumentMeta } from "@blockframes/utils/models-meta";
+import { Media, Territory } from "@blockframes/utils/static-model";
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/contract/+state/contract.firestore.ts:3]: same as above

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**